lib/inferno/apps/cli/templates/%library_name%.gemspec.tt

```diff
@@ -0,0 +1,26 @@
+Gem::Specification.new do |spec|
+  spec.name          = '<%= library_name %>'
+  spec.version       = '0.0.1'
+  spec.authors       = <%= authors %>
+  # spec.email         = ['TODO']
+  spec.date          = Time.now.utc.strftime('%Y-%m-%d')
+  spec.summary       = '<%= title_name %> Test Kit'
+  spec.description   = '<%= human_name %> Inferno test kit for FHIR'
+  # spec.homepage      = 'TODO'
+  spec.license       = 'Apache-2.0'
+  spec.add_runtime_dependency 'inferno_core', '~> 0.4.4'
```

Comment:
This should be dynamically set to the current version.

Reply:
fixed